lib/pxconfig.js

```diff
@@ -206,7 +208,8 @@ function pxDefaultConfig(PX_INTERNAL) {
         WHITELIST_EXT: [],
         BYPASS_MONITOR_HEADER: '',
         CONFIG_PATH: '',
-        ADVANCED_BLOCKING_RESPONSE: true
+        ADVANCED_BLOCKING_RESPONSE: true,
+        TELEMETRY_COMMAND_HEADER: ''
```

Comment:
We don't need this value since it will get overridden by the internal object.

Reply:
fixed